src/pages/HomePage.vue

```diff
@@ -33,11 +33,12 @@
       OverviewSkeleton
     },
     mounted () {
+      window.scrollTo(0, 0)
```

Comment:
:fire: I think you might should you wipe the threads in the store on mounted? that way you can get fresh threads.
If you don't, and you don't refresh the page you may be stuck never seeing new stuff?